Use the .so name to be a bit more resilient against minor updates of
the library.

--- Do.Addins/src/Do.Universe/ApplicationItem.cs.~1~	2008-03-14 09:26:18.000000000 +0100
+++ Do.Addins/src/Do.Universe/ApplicationItem.cs	2008-11-09 22:03:14.118065667 +0100
@@ -122,22 +122,22 @@
 			gnome_desktop_item_drop_uri_list (desktopFilePtr, uri_list, 0, IntPtr.Zero);
 		}
 		
-		[DllImport ("libgnome-desktop-2.so.2")]
+		[DllImport ("libgnome-desktop-2.so")]
 		private static extern IntPtr gnome_desktop_item_new_from_file (string file, int flags, IntPtr error);
 
-		[DllImport ("libgnome-desktop-2.so.2")]
+		[DllImport ("libgnome-desktop-2.so")]
 		private static extern void gnome_desktop_item_unref (IntPtr item);
 
-		[DllImport ("libgnome-desktop-2.so.2")]
+		[DllImport ("libgnome-desktop-2.so")]
 		private static extern IntPtr gnome_desktop_item_drop_uri_list (IntPtr item, string list, int flags, IntPtr error);
 
-		[DllImport ("libgnome-desktop-2.so.2")]
+		[DllImport ("libgnome-desktop-2.so")]
 		private static extern int gnome_desktop_item_launch (IntPtr item, IntPtr args, int flags, IntPtr error);
 
-		[DllImport ("libgnome-desktop-2.so.2")]
+		[DllImport ("libgnome-desktop-2.so")]
 		private static extern IntPtr gnome_desktop_item_get_localestring (IntPtr item, string id);
 
-		[DllImport ("libgnome-desktop-2.so.2")]
+		[DllImport ("libgnome-desktop-2.so")]
 		private static extern IntPtr gnome_desktop_item_get_string (IntPtr item, string id);
 	}
 }
